CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: modified JEXL transformations (toisostring, gettime, parseint, etc.) to return null instead of NaN when some unexpected situation occurs (#1702).
```

Comment:
```suggestion
- Fix: modified JEXL transformations (toisostring, gettime, parseint, etc.) to return null instead of NaN when some unexpected situation occurs (#1701)
```